feat: Add runbook option to gocd alert script

timmc-edx opened the pull request:
Supports https://github.com/edx/edx-arch-experiments/issues/1005

Also:

- Adjust alert message and description a bit
- Extract test environment for reuse

feanil commented:
👍

doctoryes commented:
Nit: `util.py` has two lines in it, which could be moved to `__init__.py`. I'm inclined to remove that file for now. But not a blocker.

doctoryes commented:
👍 A nit - but your call.

macdiesel commented:
@doctoryes good idea, makes the code cleaner. I'm going to make the change and merge.

timmc-edx committed:
feat: Add runbook option to gocd alert script

Supports https://github.com/edx/edx-arch-experiments/issues/1005

Also:

- Adjust alert message and description a bit
- Extract test environment for reuse
